src/System.Windows.Forms.Analyzers.CSharp/tests/UnitTests/System/Windows/Forms/Generators/ProjectFileReaderTests.FontConverter.cs

```diff
@@ -57,7 +57,9 @@ public static IEnumerable<object[]> TestConvertFormData()
         }
 
         [Theory]
+        #pragma warning disable xUnit1042 // The member referenced by the MemberData attribute returns untyped data rows
         [MemberData(nameof(TestConvertFormData))]
+        #pragma warning restore xUnit1042 // The member referenced by the MemberData attribute returns untyped data rows
```

Comment:
We typically do not want this comment twice for suppressions
```suggestion
        #pragma warning restore xUnit1042 
```

Reply:
The comment in both locations is automatically added by Roslyn's code fix

Reply:
Yep, for the most part we've been manually removing the comment on restore when adding these warnings

Reply:
Resolve this issue in new PR #11441